blog/_posts/2023-05-26-scala-3.3.0-released.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Scala 3.3.0 released!
+---
+
+![Scala 3.3]({{ site.baseurl }}/resources/img/scala-3.3-launch.png)
+
+We are thrilled to announce that after long months of work and careful testing, we have released Scala 3.3.0 LTS.
+
+## What does "LTS" mean?
+
+Scala 3.3.0 is the first Long Term Support (LTS) release. That means that it will be actively maintained for a period of at least three years. We have adopted a release model similar to the one that Java has successfully used for a long time.
+
+In the following years, there will be new minor releases (3.4, 3.5, and so on) that can bring new backward-compatible features. We are calling them Scala Next; they are equivalent to Java's feature releases. Bugfixes and usability improvements from those releases will be backported and released as 3.3.x patches. Apart from those forward and backward-compatible changes, LTS will be feature frozen.
+
+The LTS model doesn't change anything in our compatibility guarantees. Projects built with all future releases will be able to depend on any library compiled with Scala 3.3.x. This is just our standard guarantee that the newer compiler can always consume the output of the older version.
+
+LTS might be a great choice for library authors who can now receive constant bug fixes and developer experience improvements without forcing the users of the library to update the compiler version in their project.
+
+You can read more about our compatibility guarantees in [the recent blog post](https://virtuslab.com/blog/the-scala-3-compatibility-story/).
+
+## What's new in Scala 3.3.0
```

Comment:
Consider using the following explicit subsections for this section:

1. Language changes
2. Developer experience improvements
3. Standard library additions

We must make extra clear what is language and what is not language, as it confuses users too often.

Reply:
We should make things clear what is a language feature for sure, but maybe we can have some tags instead? The sections might comprise of a single point, so that seems very formal for the sake of it being formal.

Reply:
I think having a section with a single point is good. It makes it very obvious that there is actually very little that changes in the language (contrary to what we can read left and right).